src/store/modules/edit.ts

```diff
@@ -14,6 +16,32 @@ export default {
 
   getters: {
 
+    /**
+       * fetchEnrollemnt
+       *
+       * Author: pj
+       *
+       * Last updated: May 09, 2022
+       * @param data
+       */
+    fetchSingleEnrollment (
```

Comment:
signature of an action: `functionName ({...storeVariables}, payload: object)`, where at least one of the `storeVariables` should be present